### PR TITLE
Bsieber changes

### PR DESCRIPTION
**Pull request by bsieber-mozilla**

### CODE REVIEW

**Comment by mozafrank** [comment]
@bowlofstew fyi

File: .travis.yml [hunk 1]
```diff
@@ -11,18 +11,18 @@ cache:
...
     - /home/travis/.cache/pip
 before_install:
-  - pip install pipenv==2018.11.26
+  - pip install poetry
   - pip install tox
 jobs:
   include:
     - stage: Build
       script:
-        - python setup.py build
```
Comment (bowlofstew) [inline .travis.yml:20]: Do we have concern of breaking none poetry consumers?

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ language: python
 dist: xenial
 sudo: required
 python:
- - "3.7"
```
Comment (bowlofstew) [inline .travis.yml:5]: Do we just want to add a Travis CI build matrix and still support 3.7 as well as 3.8 to be backwards compatible to consumers?

**Comment by bowlofstew** [comment]
Please ignore these if we want to migrate to Python 3.8, etc.

**Comment by bsieber-mozilla** [comment]
(Closing this PR)

**Pull request closed without merging**